test/cases/testdata/netcidrmerge/test-netcidrmerge0117.yaml

```diff
@@ -0,0 +1,174 @@
+cases:
+  - note: netcidrmerge/cidr single subnet
+    modules:
+      - |
+        package generated
```

Comment:
The `generated` package was just used for migrating the existing tests. We can just use something a bit more human-friendly for new tests write (e.g., `package test`).

Reply:
It would be worthwhile to add a test case that provides IPs as inputs (not CIDRs) as the function should support both.

Reply:
Updated package name to `test` and added test cases that include IPs.